src/app/(sok)/_components/searchResult/Debug.tsx

```diff
@@ -1,10 +1,18 @@
-import React from "react";
+import React, { ReactElement } from "react";
 import { BodyShort, Box, HStack, VStack } from "@navikt/ds-react";
 import { useSearchParams } from "next/navigation";
 import { QueryNames } from "@/app/(sok)/_utils/QueryNames";
 import DebugExplain from "@/app/(sok)/_components/searchResult/DebugExplain";
+import { SurfaceColorToken } from "@navikt/ds-react/src/layout/utilities/types";
```

Comment:
Jeg kan endre dette etterpå i en egen branch, slik at vi ikke trenger å importere fra SurfaceColorToken fra /src på denne måten. Det er unansett bare debug-filene som bruker SurfaceColorToken på denne måten, og jeg kan bare droppe kompleksiteten rundt det.

Reply:
Fjernet oppsett for å definere SurfaceColorToken, brukte uansett bare en farge her uansett.